### PR TITLE
fix(#502): always use lowercase icons by filename

### PR DESCRIPTION
**Pull request by alex-courtis**
After looking at #525 it seems that #502 has regressed - lowercased filenames are not used.

Applied the lowercasing duing refresh.

Much nicer:

### CODE REVIEW

File: lua/nvim-web-devicons.lua [hunk 1]
```diff
@@ -91,9 +116,9 @@ local filetypes = {
   ["checkhealth"] = "checkhealth",
   ["commit"] = "commit_editmsg",
   ["copying"] = "copying",
-  ["gemfile"] = "gemfile$",
+  ["gemfile"] = "gemfile",
   ["lesser"] = "copying.lesser",
-  ["vagrantfile"] = "vagrantfile$",
+  ["vagrantfile"] = "vagrantfile",
```
Comment (alex-courtis) [inline lua/nvim-web-devicons.lua:121]: The `$` had no purpose and would not match. There are no more.

Comment (hasecilu) [reply lua/nvim-web-devicons.lua:121]: I think that some programs use it for backup purposes, similar with `~` character

**Review by hasecilu — APPROVED** [review]
Looks good

File: lua/nvim-web-devicons.lua (hunk 1 above)
Comment (alex-courtis) [reply lua/nvim-web-devicons.lua:121]: Oh I see. I thought it was a regex.

I'll back them out.

**Pull request merged**